docs/_components/code-formatting.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Code Formatting
+lead: >
+  Facilitates syntax highlighting that maintains visually consistentcy with the login.gov color palette
+---
+
+Intended for use by other developers when producing documentation for sites that leverage the login.gov styleguide,
+such as this one, we expose a collection of helper classes designed to be used with a
+pygments-compatible syntax highlighter. Examples of these include [peacock](https://github.com/thlorenz/peacock),
+for JavaScript, or [rouge](https://github.com/jneen/rouge), for ruby; the latter of which jekyll depends on.
+
+The class names provided map directly to a subset of classnames generated by any pygment-compatible highlighter
+during parsing, and should therefore not be used directly.
+
+The following tokens are implemented in the login.gov `styles.css` file:
+
+```css
+.c /* comment */
+.err /* error */
+.k /* keyword */
+.ch /* comment.hashbang */
+.cm /* comment.multiline */
+.cp /* comment.preproc */
+.cpf /* comment.preprocfile */
+.c1 /* comment.single */
+.cs /* comment.special */
+.gd /* generic.deleted */
+.ge /* generic.emph */
+.gr /* generic.error */
+.gh /* generic.heading */
+.gi /* generic.inserted */
+.go /* generic.output */
+.gp /* generic.prompt */
+.gs /* generic.strong */
+.gu /* generic.subheading */
+.gt /* generic.traceback */
+.kc /* keyword.constant */
+.kd /* keyword.declaration */
+.kn /* keyword.namespace */
+.kp /* keyword.pseudo */
+.kr /* keyword.reserved */
+.kt /* keyword.type */
+.m /* literal.number */
+.s /* literal.string */
+.na /* name.attribute */
+.nb /* name.builtin */
+.nc /* name.class */
+.no /* name.constant */
+.nd /* name.decorator */
+.ni /* name.entity */
+.nf /* name.function */
+.nn /* name.namespace */
+.nt /* name.tag */
+.nv /* name.variable */
+.ow /* operator.word */
+.w /* text.whitespace */
+.mb /* literal.number.bin */
+.mf /* literal.number.float */
+.mh /* literal.number.hex */
+.mi /* literal.number.integer */
+.mo /* literal.number.oct */
+.sa /* literal.string.affix */
+.sb /* literal.string.backtick */
+.sc /* literal.string.char */
+.dl /* literal.string.delimiter */
+.sd /* literal.string.doc */
+.s2 /* literal.string.double */
+.se /* literal.string.escape */
+.sh /* literal.string.heredoc */
+.si /* literal.string.interpol */
+.sx /* literal.string.other */
+.sr /* literal.string.regex */
+.s1 /* literal.string.single */
+.ss /* literal.string.symbol */
+.bp /* name.builtin.pseudo */
+.fm /* name.function.magic */
+.vc /* name.variable.class */
+.vg /* name.variable.global */
+.vi /* name.variable.instance */
+.vm /* name.variable.magic */
+.il /* literal.number.integer.long */
+```
```

Comment:
I’m concerned that this list of items will become out of date, should we make updates in the future to the scss file. We could write a test to assert these two lists are the same, but I think we might be able to drop the list instead — since these classes are written by highlighters and are a standard.

Thoughts?

Reply:
Yes, a test for this seems like overkill. I'll just add a link to the CSS if people want to inspect which classes are supported with the login.gov color palette.